.github/ISSUE_TEMPLATE/dep-golang.md

```diff
@@ -106,13 +110,13 @@ happened for the 1.20 and 1.19 Kubernetes release branches.
 
 #### After kubernetes/kubernetes (release branches) has been updated
 
-<!-- Example: https://github.com/kubernetes/release/pull/2429 -->
+<!-- Example: https://github.com/kubernetes/release/pull/2699 -->
 - [ ] `k8s-cloud-builder` image update: 
 
-<!-- Example: https://github.com/kubernetes/release/pull/2429 -->
+<!-- Example: https://github.com/kubernetes/release/pull/2699 -->
 - [ ] `k8s-ci-builder` image variants update: 
```

Comment:
Maybe we can remove those two points because it's the same PR as "After kubernetes/kubernetes (master) has been updated". I decided to still leave it for the sake of completeness.